src/wp-admin/js/dashboard.js

```diff
@@ -2,11 +2,23 @@
 var ajaxWidgets, ajaxPopulateWidgets, quickPressLoad;
 window.wp = window.wp || {};
 
+/**
+ * @summary Dashboard widget functionality.
```

Comment:
The @summary tag should only be used if The description doesn't fit on a single line. In that case, it can be used to provide a short description.